### PR TITLE
Add tts platform: service

### PR DESCRIPTION
**Pull request by al-one**
Speak message through any hass service.

```yaml
- type: custom:mini-media-player
  entity: media_player.example
  tts:
    platform: service
    data:
      service: tts.edge_tts_say
      service_data:
        cache: true

- type: custom:mini-media-player
  entity: media_player.xiaoai_speaker
  tts:
    platform: service
    data:
      service: xiaomi_miot.intelligent_speaker
      service_data:
        execute: true
        silent: true
      message_filed: text
```

### CODE REVIEW

**Comment by al-one** [comment]
I will fix the typo tomorrow.

**Comment by kalkih** [comment]
Was just about to comment on `message_filed`, but you were first 😉

File: src/components/tts.js [hunk 1]
```diff
@@ -76,7 +76,17 @@ class MiniMediaPlayerTts extends LitElement {
...
+      const [domain, service] = (config.data.service || '').split('.');
+      const field = config.data.message_filed || 'message';
+      const serviceData = {
+        [field]: message,
+        entity_id: opts.entity_id,
+        ...(config.language ? { language: opts.language } : {}),
+        ...(config.data.service_data || {}),
+      };
+      this.hass.callService(domain, service, serviceData);
+    } else this.hass.callService('tts', `${config.platform}_say`, opts);
```
Comment (kalkih) [inline src/components/tts.js:89]: It's probably time to move from `if else` cases to a `switch` here, imo.
Nothing you have to do in this PR though if you don't feel like it.

Comment (al-one) [reply src/components/tts.js:89]: I feel good. 😄

**Pull request merged**